tools/wpt/browser.py

```diff
@@ -700,6 +728,17 @@ def _latest_chromedriver_url(self, chrome_version):
     def install_webdriver_by_version(self, version, dest=None):
         if dest is None:
             dest = os.pwd
+
+        # There may be an existing chromedriver binary from a previous install.
+        # To provide a clean install experience, remove the old binary - this
+        # avoids tricky issues like unzipping over a read-only file.
+        expected_binary_path = os.path.join(dest, 'chromedriver')
```

Comment:
Actually, this is wrong. The binary name is `chromedriver.exe` on Windows. `find_binary` automatically appends `.exe` for you...

Reply:
Argh, of course. Windows! Fixed (lookup the binary instead, if it exists then delete it)